memory/numa_test.py

```diff
@@ -116,6 +117,7 @@ def test_thp_compare(self):
         """
         Test PFN's before and after offlining
         """
+        time.sleep(200)
```

Comment:
looks ugly to add sleep in test case , can we not avoid this

Reply:
ok  ,i need learn how to locked file, wait me

Reply:
@tongxintest any update here?